backend/e4h-services/ingestion-service/app/ingest/facility_template_service.py

```
@@ -49,6 +50,120 @@ def get_all_boundaries(self, request_info: RequestInfo) -> List[Boundary]:
+                lock_prefilled_rows_in_excel(
+                    file_path=output_path,
+                    sheet_name="FacilityIngestionTemplate",
+                    editable_columns=editable_columns,
+                    total_rows=len(formatted_facilities),
+                    total_columns=len(output_list),
+                    extra_append_rows=1000
+                )
+
+            logger.info(f"Successfully created template file at {output_path}")
+        except Exception as e:
+            logger.error(f"Error generating template file: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)